docs/source/app-dev/bindings-java/codegen.rst

```diff
@@ -380,3 +390,99 @@ The Java code generated for this variant is:
     public Published(Long year, String publisher) { /* ... */ }
     public Record toValue() { /* ... */ }
   }
+
+Parameterized types
+~~~~~~~~~~~~~~~~~~~
+
+A :ref:`parameterized type <daml-ref-parameterized-types>` is where the actual type is specified at instantiation as *type parameters* and not as part of type definition.
```

Comment:
I'm not sure we need to explain what parameterized types are. Especially considering that the DAML reference documentation doesn't go very deep into it either.
```suggestion
```

Reply:
Agreed with this and Gerolf's suggestion below as well.